OpenDreamClient/Interface/Controls/ControlWindow.cs

```diff
@@ -336,7 +339,11 @@ public override void SetProperty(string property, string value, bool manualWinse
         switch (property) {
             case "size":
                 if (_myWindow.osWindow is {ClydeWindow: not null}) {
-                    _myWindow.osWindow.ClydeWindow.Size = new DMFPropertySize(value).Vector;
+                    var size = new DMFPropertySize(value);
+                    var uiScale = _configurationManager.GetCVar(CVars.DisplayUIScale);
+                    size.X = (int)(size.X * uiScale);
```

Comment:
This is probably something better left for RT to do automatically, but we can just TODO that
```suggestion
                    size.X = (int)(size.X * uiScale); // TODO: RT should probably do this itself
```